### PR TITLE
Add launch stack composition logic

### PR DESCRIPTION
**Pull request by dragorosson**

### CODE REVIEW

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

File: otter/convergence/composition.py [hunk 1]
```diff
@@ -86,3 +106,20 @@ def prepare_server_launch_config(group_id, server_config, lb_descriptions):
...
+
+    :param str group_id: The group ID
+    :param PMap stack_config: The stack part of the Group's launch config,
+        as per :obj:`otter.json_schema.group_schemas.stack`.
+    """
+    # Set stack name and tag to the same thing
+    stack_config = set_in(stack_config, ('stack_name',),
+                          get_stack_tag_for_group(group_id))
+    stack_config = set_in(stack_config, ('tags',),
+                          get_stack_tag_for_group(group_id))
```
Comment (radix) [inline otter/convergence/composition.py]: we should probably append our tag to any existing 'tags' value if there is already one set in the launch config

Comment (dragorosson) [inline otter/convergence/composition.py]: Right now, there can't be because `stack_name` and `tags` are disallowed in the schema (see the JSON schema PR).

Comment (dragorosson) [inline otter/convergence/composition.py]: Added https://github.com/rackerlabs/otter/issues/1792 to track fixing this.

**Comment by radix** [comment]
+1 :)

**Pull request merged**